File: isa/dart/chimera_isa.dart
class ChimeraISA {
  static const catalogVersion = 'CHM-ISA-CATALOG-1';
  static const catalogPath = 'isa/catalog.json';
}
class InstructionRef {
  final String id, family, mnemonic, syntax, hex;
  const InstructionRef(this.id, this.family, this.mnemonic, this.syntax, this.hex);
}

File: isa/java/ChimeraISA.java
package chimera.isa;
/** Canonical ISA metadata is in ../catalog.json. */
public final class ChimeraISA {
  public static final String CATALOG_VERSION = "CHM-ISA-CATALOG-1";
  public static final String CATALOG_PATH = "isa/catalog.json";
  public record InstructionRef(String id, String family, String mnemonic, String syntax, String hex) {}
  private ChimeraISA() {}
}
